types/styled-system/index.d.ts

```diff
@@ -5,6 +5,7 @@
 //                 Justin Bennett <https://github.com/zephraph>
 //                 Christopher Pappas <https://github.com/damassi>
 //                 Eloy Durán <https://github.com/alloy>
+//                 Matthieu Vachon <https://github.com/maoueh>
```

Comment:
Are you aware that this means signing up as a maintainer and that you’ll get pinged on future PRs and tickets (once the bot starts working again, that is)?

Reply:
Yep aware, but thanks for noticing it :)